### PR TITLE
Convert client/server keying materials to local/remote ones

### PR DESCRIPTION
**Pull request by mickel8**
So far we have been returning client and server keying materials and user had to know which one to use. It depended on `client_mode` value i.e. if `client_mode` was set to `true` then `client_keying_material` was also the local one. If `client_mode` was set to `false` then the `server_keying_material` was the local one. 

In this PR we return local and remote keying material so that user don't have to know which one to use.

### CODE REVIEW

File: lib/ex_dtls.ex [hunk 1]
```diff
@@ -92,7 +93,7 @@ defmodule ExDTLS do
   def init(opts) do
     {:ok, pid} = Unifex.CNode.start_link(:native)
     :ok = Unifex.CNode.call(pid, :init, [opts[:client_mode], opts[:dtls_srtp]])
-    state = %State{cnode: pid}
+    state = %State{cnode: pid, client_mode: opts[:client_mode]}
```
Comment (mat-hek) [inline lib/ex_dtls.ex:96]: Since these options are always required, let's use dot notation, like `opts.client_mode`

**Review by mat-hek — APPROVED** [review]

File: lib/ex_dtls.ex (hunk 1 above)
Comment (mickel8) [reply lib/ex_dtls.ex:96]: It turned out that options are keyword list and dot syntax doesn't work for them

Comment (mat-hek) [reply lib/ex_dtls.ex:96]: use `Keyword.fetch!` then

Comment (mat-hek) [reply lib/ex_dtls.ex:96]: @mickel8

**Pull request merged**